standalone-metastore/metastore-catalog/pom.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>hive-standalone-metastore</artifactId>
+    <groupId>org.apache.hive</groupId>
+    <version>4.1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>hive-standalone-metastore-icecat</artifactId>
```

Comment:
`hive-standalone-metastore-iceberg-catalog` or `hive-standalone-metastore-iceberg-rest-catalog` is good to me.

Reply:
ok, agree with you the format agnostic naming since some users have the specific need to play with other tables by rest api. :)

Reply:
metastore-catalog / hive-standalone-metastore-catalog

Reply:
module name: `metastore-rest-catalog`, artifact: `hive-standalone-metastore-rest-catalog` ??
i think we should keep rest in naming

Reply:
@dengzhhu653 in your test have you built just the HMS binaries? I think it would be nice if we release just HMS binaries without the Hive stuff.